adapters/BasicAdapter/src/main/kotlin/me/gabber235/typewriter/entries/event/PlayerFishEventEntry.kt

```diff
@@ -0,0 +1,57 @@
+package me.gabber235.typewriter.entries.event
+
+import me.gabber235.typewriter.adapters.Colors
+import me.gabber235.typewriter.adapters.Entry
+import me.gabber235.typewriter.adapters.modifiers.Help
+import me.gabber235.typewriter.entry.*
+import me.gabber235.typewriter.entry.entries.EventEntry
+import me.gabber235.typewriter.utils.Item
+import org.bukkit.Location
+import org.bukkit.entity.Player
+import org.bukkit.event.player.PlayerFishEvent
+import java.util.*
+
+@Entry("on_player_fish", "When the a player caught a fish or an item", Colors.YELLOW, "mdi:fish")
+/**
+ * The `Player Fish Event` is triggered when a player catches a fish or an item.
+ *
+ * ## How could this be used?
+ *
+ * You can create custom fishing mechanics, such as catching a specific item when fishing in a specific location.
+ */
+class PlayerFishEventEntry (
+    override val id: String = "",
+    override val name: String = "",
+    override val triggers: List<Ref<TriggerableEntry>> = emptyList(),
+    @Help("The location where the fish or item was caught.")
+    val location: Optional<Location> = Optional.empty(),
```

Comment:
This doesn't make sense, the player will never be able to be on those exact coordinates. I think removing this is beter. When the user want to check if the player is in a certain region, they can use the `RegionFact` from worldguard.